repository-data/application/src/main/resources/hcm-config/namespaces/website/address.yaml

```diff
@@ -0,0 +1,213 @@
+          /buildinglocation:
+            /cluster.options:
+              jcr:primaryType: frontend:pluginconfig
+            caption: Building Location
+            field: buildinglocation
+            jcr:primaryType: frontend:plugin
+            plugin.class: org.hippoecm.frontend.editor.plugins.field.PropertyFieldPlugin
+            wicket.id: ${cluster.id}.field
+          /buildingname:
+            /cluster.options:
+              jcr:primaryType: frontend:pluginconfig
+            caption: Building Name or Number
+            field: buildingname
+            hint: ''
+            jcr:primaryType: frontend:plugin
+            plugin.class: org.hippoecm.frontend.editor.plugins.field.PropertyFieldPlugin
+            wicket.id: ${cluster.id}.field
+          /street:
+            /cluster.options:
+              jcr:primaryType: frontend:pluginconfig
+            caption: Street
+            field: street
+            hint: ''
+            jcr:primaryType: frontend:plugin
+            plugin.class: org.hippoecm.frontend.editor.plugins.field.PropertyFieldPlugin
+            wicket.id: ${cluster.id}.field
+          /area:
+            /cluster.options:
+              jcr:primaryType: frontend:pluginconfig
+            caption: Area
+            field: street
+            jcr:primaryType: frontend:plugin
+            plugin.class: org.hippoecm.frontend.editor.plugins.field.PropertyFieldPlugin
+            wicket.id: ${cluster.id}.field
+          /city:
+            /cluster.options:
+              jcr:primaryType: frontend:pluginconfig
+            caption: Town/City
+            field: city
+            hint: ''
+            jcr:primaryType: frontend:plugin
+            plugin.class: org.hippoecm.frontend.editor.plugins.field.PropertyFieldPlugin
+            wicket.id: ${cluster.id}.field
+          /county:
+            /cluster.options:
+              jcr:primaryType: frontend:pluginconfig
+            caption: County
+            field: county
+            hint: ''
+            jcr:primaryType: frontend:plugin
+            plugin.class: org.hippoecm.frontend.editor.plugins.field.PropertyFieldPlugin
+            wicket.id: ${cluster.id}.field
+          /country:
+            /cluster.options:
+              jcr:primaryType: frontend:pluginconfig
+            caption: Country
+            field: country
+            hint: ''
+            jcr:primaryType: frontend:plugin
+            plugin.class: org.hippoecm.frontend.editor.plugins.field.PropertyFieldPlugin
+            wicket.id: ${cluster.id}.field
+          /postalCode:
+            /cluster.options:
+              jcr:primaryType: frontend:pluginconfig
+            caption: Postal Code
+            field: postalCode
+            hint: ''
+            jcr:primaryType: frontend:plugin
+            plugin.class: org.hippoecm.frontend.editor.plugins.field.PropertyFieldPlugin
+            wicket.id: ${cluster.id}.field
+          /geocordinates:
```

Comment:
Should geocordinates be part of address compound, as further down it is used a separate field?